packages/ingress-nginx/versions.yaml

```
@@ -1,3 +1,4 @@
-latestVersion: "v1.9.5+1"
+latestVersion: "v1.9.6+1"
```

```suggestion
latestVersion: "v1.9.5+1"
```

We prefer to update the latestVersion via automation

---

So the bot takes the highest version in the list and updates latestVersion? Because atm the bot doesn't seem to do anyhting atm https://github.com/glasskube/packages/pull/328?

---

Yes, the bot takes the highest version. I honestly don't know why 1.9.6+1 is not in the list :eyes: